src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1049,25 +1049,22 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
 
                     if (body != null)
                     {
-                        (SyntaxNode Syntax, BoundNode Body, ExecutableCodeBinder Binder, NullableWalker.SnapshotManager SnapshotManager) forSemanticModelToUseInLambda = forSemanticModel;
+                        MethodBodySemanticModel.InitialState forSemanticModelToUseInLambda = forSemanticModel;
 
                         lazySemanticModel = new Lazy<SemanticModel>(() =>
                         {
                             var syntax = body.Syntax;
                             var semanticModel = (SyntaxTreeSemanticModel)_compilation.GetSemanticModel(syntax.SyntaxTree);
 
-                            if (forSemanticModelToUseInLambda.Syntax != null)
+                            if (forSemanticModelToUseInLambda.Syntax is { } semanticModelSyntax)
```

Comment:
>{ } [](start = 72, length = 3)

`object`

Reply:
I can't, for the same reason as the other one.

---
In reply to: [305055943](https://github.com/dotnet/roslyn/pull/37322#discussion_r305055943) [](ancestors = 305055943)